### ISSUE TITLE
E-mail address is hard-written

### ISSUE THREAD

**Issue by dunatotatos** [chiantific/Roseville#53]
To avoid scanning by spam-bots, the e-mail address should be written in JS.
Snippet:
```
<script language="JavaScript">
var username = "[your_username]";
var hostname = "[your_mail_account_domain]";
var linktext = username + "@" + hostname ;
document.write("<a href='" + "mail" + "to:" + username + "@" + hostname + "'>" + linktext + "</a>");
</script>
```

**Comment by dunatotatos** [comment]
Fixed

**Comment by dunatotatos** [comment]
Fixed in dd200e2e1fce5da77224a8eaad8cf6b15c2b570a
